torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -1334,13 +1334,14 @@ void ProcessGroupNCCL::heartbeatMonitor() {
 
 void ProcessGroupNCCL::ncclCommWatchdog() {
   try {
-    VLOG(2) << logPrefix() << "NCCL watchdog thread started!";
+    VLOG(2) << logPrefix() << "Process group watchdog thread started!";
```

Comment:
nit: change to ProcessGroupNCCL?

Reply:
Hi Will, I found the watchdog log can just discard "NCCL" keyword, and "Process group" might be enough for the log purpose. Let me know if it does not work for you.

Reply:
Well we have other process group classes (gloo, mpi) but they do not have this watchdog, that's why I left the nccl part in.

Reply:
ok, that is fine with me.